docs/local_install_skaffold.md

```diff
@@ -0,0 +1,190 @@
+
+```
+192.168.65.2 substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com
+```
+
+Running the following command will do it for you (you'll be asked for your password again)
+
+```sh
+sudo echo "192.168.65.2 substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com" > /etc/hosts
+```
+
+### 4. Install skaffold
+
+On OSX, if you have homebrew installed, just run
+
+```
+brew install skaffold
+```
+
+If you don't have homebrew installed, then install it first with the following command and then run the above command.
+
+```
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+## Running the platform
+
+Using the command line, navigate to the `hlf-k8s` folder and run:
+
+```sh
+skaffold dev
+````
+
+Then in a separate terminal, navigate to the `substra-backend` folder and run:
+
+```sh
+skaffold dev
+````
+
+Lastly, in a third terminal, navigate to the `substra-frontend` folder and once again run
+
+```sh
+skaffold dev
+```
+
+The platform will be ready once
+- the `hlf-k8s` terminal displays `INFO 003 Installed remotely response:<status:200 payload:"OK" >`:
+
+```
+[network-org-2-peer-1-hlf-k8s-chaincode-install-0-4bdd4 fabric-tools] 2019-11-14 09:14:52.070 UTC [chaincodeCmd] install -> INFO 003 Installed remotely response:<status:200 payload:"OK" >
+```
+
+- the `backend` terminal displays `"GET /readiness HTTP/1.1" 200 2` and `"GET /liveness HTTP/1.1" 200 2` for both `org-1` and `org2`:
+
+```
+[backend-org-1-substra-backend-server-7bd48859c-dr9xj substra-backend] [15/Nov/2019 08:09:29] "GET /liveness HTTP/1.1" 200 2
+[backend-org-1-substra-backend-server-7bd48859c-dr9xj substra-backend] [15/Nov/2019 08:09:36] "GET /readiness HTTP/1.1" 200 2
+[backend-org-1-substra-backend-server-7bd48859c-dr9xj substra-backend] [15/Nov/2019 08:09:29] "GET /liveness HTTP/1.1" 200 2
+[backend-org-1-substra-backend-server-7bd48859c-dr9xj substra-backend] [15/Nov/2019 08:09:36] "GET /readiness HTTP/1.1" 200 2
+```
+
+- the `frontend` terminal displays:
+
+```
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED ORIGINAL PATH: /
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED PARSED PATH: /
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] UNCACHABLE ROUTE /
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED ORIGINAL PATH: /login
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] REQUESTED PARSED PATH: /login
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] UNCACHABLE ROUTE /login
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] CHUNK NAMES [ 'user/components/index' ]
+[substra-frontend-peer-2-substra-frontend-848764cb4c-j7qjc substra-frontend] CACHING:  /login
+```
```

Comment:
It will be good to not rely on logs to ensure the platform is started? Maybe we could recommend to use `k9s` to monitor the kubernetes resources and to wait for all the resources to be either `Completed` or `Running`.

This should be always true even if we update the logs.

What do you think?

Reply:
Using k9s is a good idea, but it adds its own layer of complexity (you need to understand what are pods and services, what filters are applied etc.) Relying on the logs may not be the cleanest of options but at least it's easy.

Reply:
Sounds good to me, thanks 👍